docs/semgrep-code/editor.md

```diff
@@ -188,6 +188,31 @@ Once you've written a rule and created comment annotations, you can run your rul
 
 <EnableTurboMode />
 
+## Code search (beta)
+
+Code search allows you to test a Semgrep rule by running it against one or more GitHub repositories instead of just a few lines of test code. Its results highlight all instances of matching code in your repositories, allowing you to see whether your rule works as intended or not. This rapid feedback can help you develop more accurate and effective rules.
+
+![Code search in Semgrep Editor](/img/code-search.png)
+***Figure.*** Code search in Semgrep Editor
+
+:::info prerequisites
+* Code search is currently available to all paying customers of Semgrep Code.
+* You must have granted Semgrep code access if you would like to run code search against your repositories. Otherwise, you can run code search against public repositories.
```

Comment:
```suggestion
* You must have granted Semgrep code access if you would like to run code search against your repositories. Otherwise, you can run code search against public repositories.
```
- Can we change this to present tense? Also, does this need Semgrep Cloud Scanning or just "code access"?

Reply:
- Just "Code access" - can be granted by enabling Assistant as it creates a private Semgrep app, doesn't require ZCS, works on CI-based scans
- Needs Cloud/Managed scanning - needs more than just a private app, does not work on CI-based scans.